src/main/java/io/cdap/plugin/servicenow/source/ServiceNowMultiInputFormat.java

```diff
@@ -64,20 +67,23 @@ public static Set<ServiceNowTableInfo> setInput(Configuration jobConfig,
 
     // Depending on conf value fetch the list of fields for each table and create schema object
     // return the schema object for each table as ServiceNowTableInfo
-    Set<ServiceNowTableInfo> tableInfos = fetchTablesInfo(conf);
+    Set<ServiceNowTableInfo> tableInfos = fetchTablesInfo(conf.getConnection(), conf.getTableNames(),
+                                                          conf.getValueType(),
+                                                          conf.getStartDate(), conf.getEndDate());
 
     jobConf.setTableInfos(tableInfos.stream().collect(Collectors.toList()));
 
     return tableInfos;
   }
 
-  static Set<ServiceNowTableInfo> fetchTablesInfo(ServiceNowMultiSourceConfig conf) {
+  static Set<ServiceNowTableInfo> fetchTablesInfo(ServiceNowConnectorConfig conf, String tableName,
```

Comment:
rename tableName to tableNames
set can be renamed to tableNameSet

Reply:
Done